CONTRIBUTING.rst

```diff
@@ -208,27 +208,33 @@ You can configure the Docker-based Breeze development environment as follows:
 1. Install the latest versions of the Docker Community Edition
    and Docker Compose and add them to the PATH.
 
-2. Enter Breeze: ``./breeze``
+2. Install jq on your machine, using the following command:
+
+.. code-block:: bash
+
+  sudo apt install jq
```

Comment:
We don’t limit this guide to Ubuntu (or Debian) users, so this should say something like

```rst
Install jq on your machine. The exact command depends on the operating system (or Linux distribution) you use. For example, on Ubuntu:

.. code-block:: bash

    sudo apt install jq

Or on macOS with Homebrew:

.. code-block:: bash

    brew install jq

Reply:
Perfect! Thanks for the comment @uranusjr

I am a macOS user as well and I used to `brew install jq` to install the library. However, most of the commands in the CONTRIBUTORS_QUICK_START.rst refers to Ubuntu/Linux distribution and that's why I have limited it to the same. However, happy to update based on the review.

Reply:
Great discussion. Thanks, @potiuk and @uranusjr, for pointing out. I have updated the `CONTRIBUTING.rst` file with the example for the macOS as well, as pointed outed by @uranusjr.

The pre-commit and the GitHub checks have passed. Regarding adding the macOS prerequisites to the `CONTRIBUTORS_QUICK_START.rst`, I will send over a new PR with more robust examples for macOS. I hope my changes are looking good now. 

Would you please let me know if any more comments? I am happy to learn and address them